.buildkite/shared-pipeline-vars

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+ # This file is `source`'d before calling `buildkite-agent pipeline upload`, and can be used
+ # to set up some variables that will be interpolated in the `.yml` pipeline before uploading it.
+
+ export CI_TOOLKIT="automattic/a8c-ci-toolkit#3.4.2"
+ export TEST_COLLECTOR="test-collector#v1.10.1"
```

Comment:
Let's hope the jump fro `1.8.0` to `1.10.1` won't break anything! (It shouldn't if they respect semver…)